tests/integration/test_allclose.py

```diff
@@ -0,0 +1,284 @@
+# Copyright 2021-2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+
+import cunumeric as num
+
+SCALARS_TRUE_DEFAULT = (
+    (0, -1e-8),
+    (1e10, 1.00001e10),
+    (1 + 1j, 1 + 1.00001j),
+)
+
+SCALARS_TRUE_INF = (
+    (np.inf, np.inf),
+    (-np.inf, -np.inf),
+)
+
+
+@pytest.mark.parametrize(
+    "ab",
+    SCALARS_TRUE_DEFAULT + SCALARS_TRUE_INF,
+    ids=lambda ab: f"(a={ab[0]},b={ab[1]})",
+)
+def test_scalar_true(ab):
+    a, b = ab
+    res_np = np.allclose(a, b)
+    res_num = num.allclose(a, b)
+    assert res_np is True
+    assert res_np == res_num
```

Comment:
@bryevdv Chaining ```is``` doesn't work here as res_num is ```ndarray(True)``` while res_np is bool True. In Numpy, allclose return type is bool. In cunumeric/module.py, allclose return type is ```ndarray scalar```. Do we need to modify it to be bool?

Reply:
Ah that's unfortunate. Does this work?
```python
assert res_np is bool(res_num) is True
```
If so, I guess that could be acceptable. But otherwise any cure seems at least as complicated as the original, so probably not worth the bother. 

Up to whether to use the pattern above (assuming it works)

Reply:
It works by ```assert res_np is bool(res_num) is True```. So I use this version of chaining is. Thanks.